style.css

```diff
@@ -132,8 +141,511 @@ header {
 }
 
 .image .img1 {
+  margin-left: 32px;
   height: 300px;
   top: -30px;
   bottom: 0.36%;
   overflow: hidden;
 }
+
+/* == work section start == */
+
+.grid {
+  display: grid;
+  width: 375px;
+}
+
+.card-content {
+  display: grid;
+  place-items: center;
+  width: 375px;
+}
+
+.card-header {
+  width: 327px;
+  text-align: center;
+  margin-top: 107px;
+  margin-bottom: 10px;
+  padding: 12px 0;
+  font-size: 26px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  color: #1c1a19;
```

Comment:
- [ ] Kindly make sure only the work section container is displayed as a grid, the individual cards must be flexboxes:recycle: